README.md

```diff
@@ -25,35 +25,37 @@ TorchServe is a flexible and easy to use tool for serving PyTorch models.
 
 ## Install TorchServe
 
-1. Install Java 11
+1. Install dependencies
 
-    For Ubuntu
-    ```bash
-    sudo apt-get install openjdk-11-jdk
-    ```
-   
-   For Mac
-    ```bash
-    brew tap AdoptOpenJDK/openjdk
-    brew cask install adoptopenjdk11
-    ```
+Note: For Conda, Python 3.8 is required to run Torchserve.
```

Comment:
@maaquib: This is because the currently published conda-binaries are only compatible with Py38. It was already there in the documentation, just moved to a different place.